generators/Exercises/Generators/RestApi.cs

```diff
@@ -0,0 +1,30 @@
+﻿using Exercism.CSharp.Output;
+using Newtonsoft.Json;
+
+namespace Exercism.CSharp.Exercises.Generators
+{
+    public class RestApi : GeneratorExercise
+    {
+        protected override void UpdateTestMethod(TestMethod testMethod)
+        {
+            testMethod.UseVariableForTested = true;
```

Comment:
I think it might be useful to also use variables for the expected and input:

```
testMethod.UseVariablesForInput = true;
testMethod.UseVariableForTested = true;
testMethod.UseVariableForExpected = true;
```

Could you create a version that has the above code to see if that reads a bit better?

Reply:
Done.

Reply:
I like the effect the change has! Maybe also use `testMethod.UseVariablesForConstructorParameters = true;`?